propkatraj/propkatraj.py

```diff
@@ -5,7 +5,7 @@
 
 from __future__ import print_function, division
 
-from six import string_types, StringIO, raise_from
+from six import StringIO, raise_from
```

Comment:
We could get rid of six and just support Python >=3.8

Reply:
... but that can be another issue.

Reply:
If CI is 3.8+ then we should just make that move now.